src/screens/EditTagScreen/EditTagScreen.js

```diff
@@ -87,8 +87,23 @@ class EditTagScreen extends Component {
     navigation.navigate('Tags')
   }
 
+  refreshPermission = () => this.setState({
+    permissionVisible: false,
+  })
+
+  handlePermission = () => this.setState({
+    permissionVisible: true,
+  })
+
+  deleteTag = () => {
+    const { deleteCurrentTag, navigation, } = this.props
+
+    navigation.navigate('Tags')
```

Comment:
Use constants for screens names:

Examples:
```
navigate(NEW_WORD_SCREEN)
navigate(SCREENS.NEW_WORD)
```

Reply:
Agree